src/main/java/com/artos/mentoring/dao/TravelerDao.java

```diff
@@ -0,0 +1,12 @@
+package com.artos.mentoring.dao;
+
+import com.artos.mentoring.model.Traveler;
+import org.springframework.data.repository.CrudRepository;
+
+/**
+ * Created by Benjámin Árva on 2017. 09. 04..
+ */
+public interface TravelerDao extends CrudRepository<Traveler, Long> {
+
+    Traveler findByEmail(String email);
```

Comment:
Is email guaranteed to be a unique attribute? I'm not quite sure we should use email as a way to identify users.
If you've just pushed it as a placeholder, then ignore the comment.

Reply:
OK, ignored

Reply:
daaaamn...

Reply:
You were quick.

_that's what she said..._